src/components/FiltersPanel/FiltersPanel.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react'
+import { useFilter } from '../../contexts/filterContext'
```

Comment:
have an index.js file in the context folder, import all the context in the index, and from there export all the files, doing this will keep your imports short